build.sbt

```diff
@@ -6,7 +6,7 @@ organization := "com.kinja"
 
 version := "0.3.2" + {if (System.getProperty("JENKINS_BUILD") == null) "-SNAPSHOT" else ""}
```

Comment:
Did you consider also bumping the library's version number? Just to avoid confusion and make sure it makes it to out Nexus release repo.